.prettierignore

```diff
@@ -0,0 +1 @@
+**/.husky
```

Comment:
Alternately: can we prettify the husky files?

Reply:
Yea the ones on my local are prettified. But there are some that are git ignored. So they show after npm install in the CI

Reply:
Like .husky/_/husky.sh

Reply:
Great: anything that's checked in, we'd want Prettier-ed. And anything that's not, we don't mind either way.